From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Sun, 13 Mar 2016 19:10:11 +0100
Subject: Drop nm-applet from the session files.

nm-applet is started by the autostart script anyway, and if it's in the
session file and network-manager-gnome is not installed then the session
fails to start.
Debian-Bug: https://bugs.debian.org/818084
---
 files/usr/share/cinnamon-session/sessions/cinnamon.session   | 2 +-
 files/usr/share/cinnamon-session/sessions/cinnamon2d.session | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: cinnamon/files/usr/share/cinnamon-session/sessions/cinnamon.session
===================================================================
--- cinnamon.orig/files/usr/share/cinnamon-session/sessions/cinnamon.session	2016-11-19 13:15:06.210284188 +0100
+++ cinnamon/files/usr/share/cinnamon-session/sessions/cinnamon.session	2016-11-19 13:15:06.206284079 +0100
@@ -1,6 +1,6 @@
 [Cinnamon Session]
 Name=Cinnamon
-RequiredComponents=cinnamon;cinnamon-settings-daemon;cinnamon-screensaver;nemo-autostart;nm-applet;cinnamon-killer-daemon;
+RequiredComponents=cinnamon;cinnamon-settings-daemon;cinnamon-screensaver;nemo-autostart;cinnamon-killer-daemon;
 DesktopName=X-Cinnamon
 
 
Index: cinnamon/files/usr/share/cinnamon-session/sessions/cinnamon2d.session
===================================================================
--- cinnamon.orig/files/usr/share/cinnamon-session/sessions/cinnamon2d.session	2016-11-19 13:15:06.210284188 +0100
+++ cinnamon/files/usr/share/cinnamon-session/sessions/cinnamon2d.session	2016-11-19 13:15:06.206284079 +0100
@@ -1,6 +1,6 @@
 [Cinnamon Session]
 Name=Cinnamon (Software Rendering)
-RequiredComponents=cinnamon2d;cinnamon-settings-daemon;cinnamon-screensaver;nemo-autostart;nm-applet;cinnamon-killer-daemon;
+RequiredComponents=cinnamon2d;cinnamon-settings-daemon;cinnamon-screensaver;nemo-autostart;cinnamon-killer-daemon;
 DesktopName=X-Cinnamon
 
 
